addons/web/static/tests/core/popover/popover_tests.js

```diff
@@ -158,3 +158,72 @@ QUnit.test("reposition popover should properly change classNames", async (assert
     );
     assert.strictEqual(arrow.className, "popover-arrow top-auto");
 });
+
+QUnit.test("arrow height is taken into account when positioning popover", async (assert) => {
+    // Force some style, to make this test independent of screen size
+    const container = document.createElement("div");
+    container.id = "container";
+    container.style.backgroundColor = "pink";
+    container.style.height = "450px";
+    container.style.width = "450px";
+    container.style.display = "flex";
+    container.style.alignItems = "center";
+    container.style.justifyContent = "center";
+    popoverTarget.style.backgroundColor = "yellow";
+    popoverTarget.style.height = "50px";
+    popoverTarget.style.width = "50px";
+    container.appendChild(popoverTarget);
+    const sheet = document.createElement("style");
+    sheet.textContent = `
+        [role=tooltip] {
+            background-color: cyan;
+            height: 100px;
+            width: 100px;
+        }
+        .popover-arrow {
+            height: 20px!important;
+        }
+    `;
+    fixture.appendChild(container);
+    document.head.appendChild(sheet);
+    registerCleanup(() => {
+        container.remove();
+        sheet.remove();
+    });
+
+    const TestPopover = class extends Popover {
+        onPositioned(el, position) {
+            super.onPositioned(el, position);
```

Comment:
Maybe an "assertStep" is needed, in order to be sure that the test doesn't pass just because onPositioned is never called? Maybe overkill :shrug: